### DISCUSSION TITLE
Which FASTA file to use? reviewed vs unreviewed vs canonical vs isoforms

### DISCUSSION THREAD

**Discussion by luizalmeida93** [vdemichev/DiaNN#740] [Q&A] [answered]
Hi all,

I am switching a few experiments from DDA to DIA, and I was wondering which FASTA file to use in the library-free mode.

When downloading proteomes from UniProt, we can choose what proteins are included: reviewed, unreviewed, canonical and isoforms. The most comprehensive database contains all of the above, while the most conservative would be only "reviewed + canonical". In one of the DIA-NN papers, it seem the conservative approach was used, but I didn't see any mention of the FASTA composition on the summary page.

I made a quick and super simple comparison using 12 files/samples from serum. Samples were acquired in a 2-hours gradient. I used the report.gg file and it yielded the following results:

**Reviewed + unreviewed + canonical + isoform:**
- Total of 424 proteins
- 150 proteins (35%) quantified in all samples
- 229 proteins (54%) quantified in 90% of the samples 

**Reviewed + canonical:**
- Total of 354 proteins
- 150 proteins (35%) quantified in all samples
- 229 proteins (54%) quantified in 90% of the samples 

**Human plasma proteome:**
- Total of 416 proteins
- 156 proteins (38%) quantified in all samples
- 225 proteins (54%) quantified in 90% of the samples 

The coverage doesn't seem to change substantially, but the comprehensive approach does yield more proteins. I know this is a very basic comparison, but am I missing something here? Is there anything else you would compare?

For now, I have only data from serum samples, but the same question would apply to tissue and cell culture.

**Comment by vdemichev** [comment]
Not entirely sure how you count proteins (which report column, what's being counted)?
I recommend to use the main report only.

**Reply by luizalmeida93** [reply]
Thank you for your answer, Vadim.
I used the report.gg file and counted the number of rows.

If I count the number of **unique** IDs in the report file, I have the following:

**Reviewed + unreviewed + canonical + isoform:**
- Protein.Group: 789 total (435 with Lib.PG.Q.Value <= 0.01)
- Protein.Ids: 1497 total (1140 with Lib.PG.Q.Value <= 0.01)
- Genes: 765 total (424 with Lib.PG.Q.Value <= 0.01)

**Reviewed + canonical:**
- Protein.Group: 703 total (416 with Lib.PG.Q.Value <= 0.01)
- Protein.Ids: 1004 total (599 with Lib.PG.Q.Value <= 0.01)
- Genes: 698 total (416 with Lib.PG.Q.Value <= 0.01)

**Human plasma proteome:**
- Protein.Group: 694 total (416 with Lib.PG.Q.Value <= 0.01)
- Protein.Ids: 882 total (599 with Lib.PG.Q.Value <= 0.01)
- Genes: 694 total (416 with Lib.PG.Q.Value <= 0.01)

Looking at unique values like this seems like an oversimplified way of approaching this question. At first, I would prefer using the comprehensive FASTA (reviewed + unreviewed + canonical + isoform) because it does have a higher protein count, but do I risk anything using such a large FASTA file?

**Reply by luizalmeida93** [reply]
Hi @vdemichev. I am following up on this question. Do you have any recommendations on the FASTA file to be used? Any drawbacks to using the full FASTA (reviewed + unreviewed + canonical + isoforms), or is it better to use a more conservative FASTA with only "reviewed + canonical" proteins?

**Reply by vdemichev** [reply, marked as answer]
Two points:
- Probably OK in any case
- We usually use canonical (including both reviewed and unreviewed), unless specifically interested in isoforms

**Reply by luizalmeida93** [reply]
Perfect! I was mostly unsure about the unreviewed proteins as they expand the search space by a significant margin, but if you are using it that is great!
Thank you for your time and for such an excellent software.
